styles.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: rgb(151, 55, 189);
+}
```

Comment:
- [ ] [**OPTIONAL**] Great job :small_airplane:  in styling your `h1` tag using the `header` class, I will suggest you in the future to use a class name that is different from existing tag name like `header` `section` etc :+1:

Reply:
Hello @iambenkis ,

I really appreciate your suggestion. I will keep it in my to improve my code next time.

Thank you for the kind review 😄